_data/es.yml

```diff
@@ -328,7 +328,7 @@ markets:
 
 stats:
   hNetworkStats: Estado de red
-  pSeeMarkets: "<p>Ver la <a href='localhost:4001/markets'>página de Mercados</a> para el volumen por moneda.</p>"
+  pSeeMarkets: "<p>Ver la <a href='https://bisq.network/markets'>página de Mercados</a> para el volumen por moneda.</p>"
```

Comment:
Add lang code to the markets' page hyper links
https://bisq.network/markets
https://bisq.network/es/markets

Reply:
@huey735 good catch
```suggestion
  pSeeMarkets: "<p>Ver la <a href='https://bisq.network/es/markets'>página de Mercados</a> para el volumen por moneda.</p>"
```